File: Beginner/1036 (Bhaskara's Formula)/1036.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;
import java.text.DecimalFormat;
public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.00000");
        double a = ler.nextDouble();
        double b = ler.nextDouble();
        double c = ler.nextDouble();
        double delta = Math.pow(b, 2) -4 * (a*c);
        double result;
        if (a<=0 || delta < 0){
            System.out.println("Impossivel calcular");
        }
        else{
            result =((-b) + Math.sqrt(delta))/(2*a);
            System.out.println("R1 = " + df.format(result));
            result =((-b) - Math.sqrt(delta))/(2*a);
            System.out.println("R2 = " + df.format(result));
        }
    }
 
}

File: Beginner/1036 (Bhaskara's Formula)/1036.cpp
#include <iostream>
#include <stdio.h>
#include <math.h>
using namespace std;
int main()
{
    double A,B,C,delta;
    cin >> A >> B >> C;
    delta= pow(B,2.0) -4 * A * C;
    if (A<=0 || delta < 0){
        cout << "Impossivel calcular" << endl;
    }
    else{
        printf("R1 = %.5f\n", ((-B) + sqrt(delta))/(2*A));
        printf("R2 = %.5f\n", ((-B)- sqrt(delta))/(2*A));
    }

    return 0;
}